themes/base.theme.bash

```diff
@@ -542,3 +544,19 @@ function _save-and-reload-history() {
   local autosave=${1:-0}
   [[ $autosave -eq 1 ]] && history -a && history -c && history -r
 }
+
+function prompt_colorscheme {
+   [[ -z "${BASH_IT_COLORSCHEME}" ]] && return
+
+   local -a colorscheme_locations=(
+     "${HOME}/.${BASH_IT_COLORSCHEME}.colorscheme.bash"
```

Comment:
Huh l think this order should be reversed, so that a custom theme overwrites the built in one

Reply:
Yes, custom should overwrite built-in - but doesn't it work like that in your implementation?

Reply:
Since the loop does not have a `break`, the color scheme file in the bash-it folder with the same name would overwrite the custom one.